src/core/component.js

```diff
@@ -203,6 +203,7 @@ Component.prototype = {
     if (!this.initialized) {
       this.init();
       this.initialized = true;
+      if (el.isPlaying) { this.play(); }
```

Comment:
Sort of confusing since seems like it should read `should be playing` rather than `is playing`. A comment might help.